M2/Macaulay2/packages/Macaulay2Doc/functions/floor-doc.m2

```diff
@@ -2,14 +2,12 @@
 --- author(s): MES
 --- notes: just added example
 
-undocumented{(floor, ZZ)}
-
 document { 
-     Key => {floor,(floor,RR),(floor, QQ),(floor, CC)},
+     Key => {floor,(floor,Number)},
```

Comment:
Good change.  I suggest having separate nodes for the simple keys "floor" and "ceiling".  Here is the way
the node for "prune" looks:

```
document { 
     Key => prune,
     Headline => "prune, e.g., compute a minimal presentation",
     SeeAlso => {minimalPresentation}
     }
```

The idea is that if other methods are going to be added, this simple node can be an umbrella node for
all of them.